src/tests/reducers/likeReducer.test.js

```diff
@@ -0,0 +1,114 @@
+import * as types from '../../redux/constants/articles';
+import likeReducer from '../../redux/reducer/article/likes';
+
+const initialState = {
+  articleLiked: false,
+  articleDisliked: false,
+};
+describe.only('Like Reducer', () => {
+  // it('it should like an article', (done) => {
```

Comment:
Nice job please remove the commented lines of code if you no longer need them.

Reply:
Okay. I will remove them and update the PR